libstb/secvar/secvar_devtree.c

```diff
@@ -64,3 +64,18 @@ void secvar_set_update_status(uint64_t val)
 	dt_add_property_u64(secvar_node, "update-status", val);
 }
 
+bool secvar_check_physical_presence(void)
+{
+	struct dt_node *secureboot;
+
+	secureboot = dt_find_by_path(dt_root, "ibm,secureboot");
+	if (!secureboot)
+		return false;
+
+	if (dt_find_property(secureboot, "clear-os-keys")
+			|| dt_find_property(secureboot, "clear-all-keys")
+			|| dt_find_property(secureboot, "clear-mfg-keys"))
+		return true;
+
+	return false;
+}
```

Comment:
Since this function is more to read the system state by reading device-tree values.. it looks more like a secvar_util function rather than secvar_devtree. I think secvar_devtree is more like providing easy interface to manipulate or modify device-tree..

Reply:
All the `secvar_util` functions operate on secvars, or lists of secvars. This function would be just as inappropriate there as well.